wapitiCore/net/scope.py

```diff
@@ -46,29 +46,37 @@ def name(self) -> str:
         return self._scope
 
     def check(self, resource: Union[Request, str]) -> bool:
+
+        checked = None
+
         if not resource:
-            return False
+            checked = False
```

Comment:
I think it should be a `return` here otherwise the scope `punk` may return `True` when a resource is invalid